src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,71 @@
+.menu {
+  background-color: #e5e5e5;
+
+  &__item {
+    margin-bottom: 24px;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  &__top {
+    margin-bottom: 44px;
+  }
+
+  &__link {
+    display: inline-block;
+    padding-bottom: 8px;
+    border-bottom: 1px solid transparent;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
```

Comment:
what is the default `font-style` of font?

Reply:
normal :)